gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java

```diff
@@ -230,6 +232,10 @@ private static void loadGenericJobConfigsRecursive(List<Properties> jobConfigs,
               Charsets.UTF_8)) {
             propertiesConfiguration.load(inputStreamReader);
             jobProps.putAll(ConfigurationConverter.getProperties(propertiesConfiguration));
+
+            jobProps =
+                (new SimpleGeneralJobTemplate(ConfigurationKeys.JOB_TEMPLATE_PATH, jobProps)).getResolvedConfig();
```

Comment:
shouldn't you be passing in `jobProps.getProperty(ConfigurationKeys.JOB_TEMPLATE_PATH)` instead?